Header header

sensor_msgs/PointCloud2 full_cloud      # full cloud
sensor_msgs/PointCloud2 corner_cloud    # extracted corner feature
sensor_msgs/PointCloud2 surface_cloud   # extracted surface feature
